mintpy/view.py

```diff
@@ -422,36 +425,37 @@ def plot_slice(ax, data, metadata, inps=None):
             vprint('plot in Lat/Lon coordinate')
             vprint('map projection: '+inps.map_projection)
             vprint('boundary database resolution: '+inps.resolution)
-            if inps.map_projection in ['cyl', 'merc', 'mill', 'cea', 'gall']:
-                m = pp.BasemapExt(llcrnrlon=inps.geo_box[0], llcrnrlat=inps.geo_box[3],
-                                  urcrnrlon=inps.geo_box[2], urcrnrlat=inps.geo_box[1],
-                                  projection=inps.map_projection,
-                                  resolution=inps.resolution, area_thresh=1.,
-                                  suppress_ticks=False, ax=ax)
-            elif inps.map_projection in ['ortho']:
-                m = pp.BasemapExt(lon_0=(inps.geo_box[0]+inps.geo_box[2])/2.0,
-                                  lat_0=(inps.geo_box[3]+inps.geo_box[1])/2.0,
-                                  projection=inps.map_projection,
-                                  resolution=inps.resolution, area_thresh=1.,
-                                  suppress_ticks=False, ax=ax)
-            else:
-                m = pp.BasemapExt(lon_0=(inps.geo_box[0]+inps.geo_box[2])/2.0,
-                                  lat_0=(inps.geo_box[3]+inps.geo_box[1])/2.0,
-                                  llcrnrlon=inps.geo_box[0], llcrnrlat=inps.geo_box[3],
-                                  urcrnrlon=inps.geo_box[2], urcrnrlat=inps.geo_box[1],
-                                  projection=inps.map_projection,
-                                  resolution=inps.resolution, area_thresh=1.,
-                                  suppress_ticks=False, ax=ax)
-
-            # Draw coastline
-            if inps.coastline:
+            m=ax
```

Comment:
Variable `m` is an awkward usage from my time, to distinguish BasemapExt object from matplotlib.pyplot.axes object. It seems like it's not needed anymore, right? If so, please remove it.

Reply:
Yah. We can remove it. It was more to keep with the previous convention.